src/test/kotlin/HttpResponseBuilderTest.kt

```diff
@@ -3,6 +3,7 @@ import kotlin.test.*
 import response.HttpResponseBuilder
 import httpstatus.HttpStatus
 import org.json.JSONObject
+import kotlin.math.exp
```

Comment:
Why is this import added?

Reply:
I believe it was auto imported. I have to figure out how to turn off the auto-import feature in Intellij